modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -70,6 +70,7 @@ def to_internal
       resolve_special_issue_mappings!
       resolve_homelessness_situation_type_mappings!
       resolve_homelessness_risk_situation_type_mappings!
+      cast_claim_date!
```

Comment:
Probably ought to move this up so that it occurs right after the other line dealing with `claimDate`.  (i.e. move this to line #60)